code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -76,7 +76,8 @@
 
 // Reuse the cache/code from stools, todo maybe unify.
 /obj/structure/bed/update_icon()
-	generate_strings()
+	if(!istype(src, /obj/structure/bed/stool/chair/office/wheelchair))
+		generate_strings()
```

Comment:
This is a horrible pattern, you should instead override `/obj/structure/bed/stool/chair/office/wheelchair/generate_strings()` to not do anything (flat return)